### PR TITLE
Feat/add xocolatl new collats wsteth maticx

### PR DESCRIPTION
**Pull request by DaigaroCota**
LaDAO-Xocolatl UPDATES
- Noticed new `coreAssets.json` and fixed WMATIC address. It was surprising that the main WMATIC address is not referenced.
- Added Lido - wsteth asset in  `coreAssets.json` 
- Added Stader - maticx asset in  `coreAssets.json` 
- Added reserve addresses for wsteth and maticx from Xocolatl protocol.

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/ladao-xocolatl exports TVL: 
        
 
 
    bsc                       69.24580000000002
    polygon                   14.04 k
    
    total                    14.11 k

**Comment by DaigaroCota** [comment]
All updated assets are for Polygon (Pos)

File: projects/helper/coreAssets.json [hunk 1]
```diff
@@ -106,15 +106,17 @@
     "aBNBb": "0xbb1aa6e59e5163d8722a122cd66eba614b59df0d"
   },
   "polygon": {
-    "WMATIC": "0xfd28c7cea3c50a060cb4c0059e453c6d4dd9829d",
+    "WMATIC": "0x0d500B1d8E8eF31E21C99d1Db9A6444d3ADf1270",
```
Comment (g1nt0ki) [inline projects/helper/coreAssets.json:109]: please revert this change, it is going to break other adapters

Comment (DaigaroCota) [reply projects/helper/coreAssets.json]: Reverted WMATIC back to original address and added WMATIC_2.

**Comment by llamatester** [comment]
The adapter at projects/ladao-xocolatl exports TVL: 
        
 
 
    polygon                   14.13 k
    bsc                       69.72975
    
    total                    14.20 k

File: projects/helper/coreAssets.json (hunk 1 above)
Comment (g1nt0ki) [reply projects/helper/coreAssets.json]: thanks
